### PR TITLE
Fix/#1773

### PR DESCRIPTION
**Pull request by hexyls**
related https://github.com/protofire/omen-exchange/issues/1773

contains small fixes:
- resets tweet messages
- fixes market history graph when using relay
- fixes relay max withdraw 
- only fetches bridge allowance when on mainnet

### CODE REVIEW

**Review by pimato — APPROVED** [review]

**Review by Mi-Lan — APPROVED** [review]
Gj finding these ones!

**Review by KadenZipfel — APPROVED** [review]
🔥

File: app/src/services/cpk.ts [hunk 1]
```diff
@@ -1360,11 +1360,13 @@ class CPKService {
         // get mainnet relay signer
         const to = await this.cpk.ethLibAdapter.signer.signer.getAddress()
 
+        const withdrawAmount = amount.sub(RELAY_FEE)
```
Comment (KadenZipfel) [inline app/src/services/cpk.ts:1363]: It looks like subtracting the relay fee causes problems with the minimum withdraw amount. If a user tries to withdraw the minimum 10 DAI, it's really only withdrawing 9.999, which causes the transaction to revert.

**Review by KadenZipfel — CHANGES_REQUESTED** [review]

File: app/src/services/cpk.ts (hunk 1 above)
Comment (KadenZipfel) [reply app/src/services/cpk.ts:1363]: As a note, the currently discussed solution is to only charge a relay fee if the amount to withdraw is not exactly 10 xDai, but seems like the relayer infrastructure doesn't allow omitting a fee: https://github.com/hexyls/omen-relay/blob/10429f7cbe24a52b3e77e42439dea540c5fb64c2/api/index.js#L35

**Review by KadenZipfel — APPROVED** [review]
🚀

**Comment by Mi-Lan** [comment]
Works! Heres tx https://blockscout.com/poa/xdai/tx/0xb2daa1a3c55057596c59c0b494141e46cf72f55467cb5d2d3a94f6fb0fd0dcdf

**Pull request merged**